src/azure/azureMEssageDisplayer.ts

```diff
@@ -0,0 +1,10 @@
+import { MessageDisplayer } from '@cssuh/ads-adal-library';
+
+export class AzureMessageDisplayer implements MessageDisplayer {
+    displayInfoMessage(msg: string): Promise<void> {
+        return;
```

Comment:
Not implementing these? At least make sure they're async so they return a promise